sphinx_compas_theme/compaspkg/layout.html

```diff
@@ -231,6 +240,20 @@ <h4 class="pb-3 footer-text">COMPAS</h4>
     }).addClass("active");
 
   });
+
+
+  let url = $("#doc_versions").data("url");
+  let docs_site = $("#doc_versions").data("docs");
+
+  if (url) {
+    $.get(url, res=>{
+      let lines = res.match(/[^\r\n]+/g);
+      lines.forEach(version => {
+        $("#doc_version_list").append(`<a class="dropdown-item" href="${docs_site+version}">${version}</a>`);
```

Comment:
Template strings are "relatively" new, and considering here the usage is very simple, it might be best to just do good ol' string concatenation instead.

Reply:
Just changed to older JS style in the new commit